chains/solana/types.go

```diff
@@ -0,0 +1,27 @@
+package solana
+
```

Comment:
Could you add a comment here on why we need this/who is using? Thanks!

Reply:
done!